spec/lib/tasks/taxonomy/health_checks_spec.rb

```diff
@@ -1,20 +1,18 @@
 require "rails_helper"
-require "gds_api/test_helpers/content_store"
 
 RSpec.describe "taxonomy:health_checks" do
-  include ::GdsApi::TestHelpers::ContentStore
-  include PublishingApiHelper
-  include ContentItemHelper
   include RakeTaskHelper
   include ActiveJob::TestHelper
 
   it "queues the expected jobs" do
     ActiveJob::Base.queue_adapter = :test
 
-    expect {
-      rake "taxonomy:health_checks"
-    }.to change(TaxonomyHealth::MaximumDepthMetric.jobs, :size).by(1)
-    .and change(TaxonomyHealth::ContentCountMetric.jobs, :size).by(1)
-    .and change(TaxonomyHealth::ChildTaxonCountMetric.jobs, :size).by(1)
+    Sidekiq::Testing.fake! do
```

Comment:
Ah this is cool, I didn't know about this API!

Reply:
💯 
I'd started off by running the test with `Sidekiq::Testing::inline!` and adding stubs, but what this needed was for things to _not_ be inline so they could go on the queue (which is what the tests want to measure).